egs/librispeech/ASR/pruned_transducer_stateless7/export-onnx.py

```diff
@@ -581,7 +581,7 @@ def main():
     quantize_dynamic(
         model_input=decoder_filename,
         model_output=decoder_filename_int8,
-        op_types_to_quantize=["MatMul"],
+        op_types_to_quantize=["MatMul", "Gather"],
```

Comment:
Could you also update
https://github.com/k2-fsa/icefall/blob/27ae1d83ba7461bdda97eb305d84dd480eca586f/egs/librispeech/ASR/zipformer/export-onnx.py#L604

https://github.com/k2-fsa/icefall/blob/27ae1d83ba7461bdda97eb305d84dd480eca586f/egs/librispeech/ASR/zipformer/export-onnx-streaming.py#L759

Reply:
Noted! https://github.com/k2-fsa/icefall/pull/1152 has been applied to these files.